apps/avifenc.c

```diff
@@ -2019,17 +2019,11 @@ int main(int argc, char * argv[])
         }
         // Matrix coefficients.
         if (settings.cicpExplicitlySet) {
-            avifBool incompatibleMC = (settings.matrixCoefficients != AVIF_MATRIX_COEFFICIENTS_IDENTITY);
-#if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
-            incompatibleMC &= (settings.matrixCoefficients != AVIF_MATRIX_COEFFICIENTS_YCGCO_RE &&
-                               settings.matrixCoefficients != AVIF_MATRIX_COEFFICIENTS_YCGCO_RO);
-#endif
+            avifBool incompatibleMC = (settings.matrixCoefficients != AVIF_MATRIX_COEFFICIENTS_IDENTITY) &&
+                                      (settings.matrixCoefficients != AVIF_MATRIX_COEFFICIENTS_YCGCO_RE &&
+                                       settings.matrixCoefficients != AVIF_MATRIX_COEFFICIENTS_YCGCO_RO);
```

Comment:
Optional: The `incompatibleMC` variable is no longer necessary.

Reply:
Done